book/invalidation.md

```diff
@@ -167,6 +167,79 @@ API!
 
 [editing-hates-you]: https://lord.io/text-editing-hates-you-too/
 
+
+Why Invalidation?
+=================
+
+Fundamentally, the reason editing this page is slow in our browser is
+that this page is pretty big.[^other-reasons] After all, it's not
```

Comment:
```suggestion
it is pretty big.[^other-reasons] After all, it's not
```

Reply:
I did something different but I think still avoids the duplication.